### PR TITLE
Admin design

### PR DESCRIPTION
**Pull request by decabeza**
What
====
- We have started to work on a big redesign for Admin section. The objective of these changes are improve the admin panel, and makes a section more clean, comfy and easy to use.

This is the first PR of many more coming next weeks 🎉 

How
===
- Cleans the header: makes title smaller and remove logo.
- Replaces red (_a little aggressive_) color to a fresh and calm blue tone.
- Moves **stats** and **settings** button to header.
- Moves "back to org" link to dashboard/index.
- Removes texts untranslated (and outdated) of dashboard/index page.
- Updates spec (admin section has no longer a header logo).

Next steps
===

This are some improvements we want to include on next PRs:

- Add icons to all sidebar sections.
- Improve sidebar section (some expand sections are not working correctly)
- Allow to show/hide sidebar (already done for mobile version)
- Add coherence and consistency on all elements (tables, filters, forms, buttons...)
- Add little help tips on each sections.
- Reorganize stats: #1596 
- Improve settings page:  #1597

And much more! 💪 

Screenshots
===========
awesome_admin_consul

Warnings
========
- The **stats** and **settings** buttons on dashboard/index now are on the header navigation.

**BEFORE**
buttons

**AFTER**
header

### CODE REVIEW

File: app/views/admin/shared/_admin_shortcuts.html.erb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<li>
+  <%= link_to admin_stats_path, title: t("admin.menu.stats") do %>
+    <span class="icon-stats"></span>
+
```
Comment (bertocq) [inline app/views/admin/shared/_admin_shortcuts.html.erb]: Extra line brake :trollface:

File: app/views/layouts/_admin_header.html.erb [hunk 2]
```diff
@@ -2,34 +2,31 @@
...
-      <div id="responsive-menu">
+
         <div class="top-bar-title">
           <h1>
-            <%= link_to namespaced_root_path, class: "hide-for-small-only" do %>
-              <%= image_tag(image_path_for('logo_header.png'), class: 'hide-for-small-only float-left', size: '80x80', alt: t("layouts.header.logo")) %>
+            <%= link_to namespaced_root_path do %>
               <%= setting['org_name'] %>
-              &nbsp;|&nbsp;<%= namespaced_header_title %>
+              <br><small><%= namespaced_header_title %></small>
```
Comment (bertocq) [inline app/views/layouts/_admin_header.html.erb]: Nice catch!

File: app/views/admin/dashboard/index.html.erb [hunk 3]
```diff
@@ -1,99 +1,9 @@
...
-<% end %>
+<div class="float-right">
+  <%= link_to root_path do %>
+    <%= t("admin.dashboard.index.back") + " " + setting['org_name'] %>
+  <% end %>
+</div>
 
 <h2 class="inline-block"><%= t("admin.dashboard.index.title") %></h2>
 
-<p>Desde aquí puedes administrar el sistema, a través de las siguientes acciones:</p>
```
Comment (bertocq) [inline app/views/admin/dashboard/index.html.erb]: Nice clean of hardcoded stuff!

File: app/views/admin/shared/_admin_shortcuts.html.erb (hunk 1 above)
Comment (decabeza) [reply app/views/admin/shared/_admin_shortcuts.html.erb]: fixed! 😱

**Comment by deivid-rodriguez** [comment]
Hi!

So the explanations for each section are coming back in a better form, right? It would be this bullet point?

> * Add little help tips on each sections.

Just wanted to point out that when I tried consul the other day, I loved this tiny feature because it felt like having docs inside the application itself. To me, it was one of the strongest points about the admin panel.

Finally, the new design and color _definitely_ look better! :muscle:

**Comment by voodoorai2000** [comment]
@deivid-rodriguez Yes, section explanations are coming back in a nice form :)

**Pull request merged**